apps/svelte.dev/content/docs/svelte/98-reference/20-svelte.md

```diff
@@ -556,6 +556,36 @@ type ComponentEvents<Comp extends SvelteComponent> =
 
 </div>
 
+## ComponentExports
+
+Convenience type to get the properties that given component exports.
+
+Example: Typing the `bind:this` for a component named `MyComponent`
+```
+<script lang="ts">
+	import MyComponent from '$lib/component';
+	let component: ComponentExports<typeof MyComponent> | undefined = undefined;
```

Comment:
wait is `ComponentExports` just `ReturnType`? it doesn't allow us to get rid of the `typeof`? is that really worthwhile?

I could have sworn that at one point we had this working properly so that you can do `let component: MyComponent`, by exporting both the value and the type separately, what happened to that?